api/DOMException.json

```diff
@@ -101,7 +101,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMException/code",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
Hmm, this one's a bit odd.  When testing in a few different Chrome versions (1, 26, and 42), the `code` variable is `undefined` and not present in the prototype.

<img width="546" alt="Screen Shot 2020-08-28 at 08 20 24" src="https://user-images.githubusercontent.com/5179191/91584500-97db5c80-e907-11ea-88fb-5e4e6adb3479.png">
<img width="679" alt="Screen Shot 2020-08-28 at 08 19 26" src="https://user-images.githubusercontent.com/5179191/91584501-990c8980-e907-11ea-9a68-78a529701d02.png">
<img width="468" alt="Screen Shot 2020-08-28 at 08 09 51" src="https://user-images.githubusercontent.com/5179191/91584506-9a3db680-e907-11ea-8259-4a67867b8ba8.png">
<img width="470" alt="Screen Shot 2020-08-28 at 07 43 05" src="https://user-images.githubusercontent.com/5179191/91584512-9ad64d00-e907-11ea-8d3a-9b9cb2b06216.png">

However, the attribute __is__ present in Chrome's source since the initial implementation of `DOMException` -- see https://source.chromium.org/chromium/chromium/src/+/8d75ff802c1d2d66f5224851415b955343a18c42:third_party/WebKit/WebCore/dom/DOMCoreException.idl;bpv=0.  I'm wondering if it's `undefined` upon initialization, one of those cases that makes testing quite frustrating...

Reply:
Ah, yeah.  When working on the `Element` API, I found that until Chrome 43, properties were sometimes left as `undefined`, which was probably the case here.  Not sure how to initiate a `DOMException`, will have to research that, and then define tests/overrides in the collector...

Reply:
Reverting these changes.  We can come back to them another time.

Reply:
Some more details about what happened in comments in https://github.com/mdittmer/mdn-confluence/commit/5e173b18caedd5dd4aec38675712d514c9fe4784. It should affect all attributes on prototypes, with the exception of global scopes like Window, where attributes are still on the instance and Window.prototype, which is correct per Web IDL.

Reply:
This still needs reverting.